### PR TITLE
Feature/user profile route

### PR DESCRIPTION
**Pull request by Willcowen**
Added a route to user/:id, on front end - back end was working already.

Created a new component in src/components/users/userprofile, this will display the profile of the user based on the /:id passed in the URL.

### CODE REVIEW

File: src/components/users/userProfile/UserProfile.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    github_url: "",
+  };
+  const [profile, setProfile] = useState(initialProfileState);
+  const { id } = useParams();
+
+  useEffect(() => {
+      foundProfile()
+  }, [])
+
+  const foundProfile = () => {
```
Comment (phillipdunn) [inline src/components/users/userProfile/UserProfile.js:21]: maybe a more intuative name could be used?

**Review by phillipdunn — DISMISSED** [review]
LGTM - one none blocking comment

**Comment by Willcowen** [comment]
image

File: src/components/users/userProfile/UserProfile.js (hunk 1 above)
Comment (Willcowen) [reply src/components/users/userProfile/UserProfile.js]: I have changed the function name to handleFindProfile, hopefully this is more suitable!

**Review by nicopicchio — APPROVED** [review]

**Review by JasonKundi — APPROVED** [review]
Looks good to me Will

**Pull request merged**